.github/workflows/releaseWithCoreBundle.yml

```diff
@@ -0,0 +1,16 @@
+name: publish source-deploy-retrieve-bundle
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
you might also want to parameterize that npm token in case you ever have something outside `forcedotcom` that wants to use your workflows.

It's a bit more useful than `secrets: inherit` and then keeps all the secrets out that it doesn't need.